### DISCUSSION TITLE
Dependency Dashboard - Checkboxes not triggering MR creation

### DISCUSSION THREAD

**Discussion by andorfermichael** [renovatebot/renovate#10525] [Request Help] [answered]
I have two questions regarding the Dependency Dashboard:

1) I'm on a self-hosted GitLab instance, and it seems to me that the checkboxes are not working. If I check them, no MR is created. Do I have to do an additional setup/config to get this working?
2) Why does the dependency dashboard include approvals (MR creation) for peer dependencies, but if I let the MRs be created without dashboard approval no peer dependency MRs are created?

**Comment by viceice** [comment, marked as answer]
1. How do you run renovate? If it's on gitlab pipeline with shedule, then you need to wait for next schedule or manually trigger a run.

2. I think you have some wrong config, so please share.

**Reply by andorfermichael** [reply]
1. ok, good to know
2. After re-run it worked

Thanks.
